### PR TITLE
MM-25946 prompt sys admins if they are running server versions below 5.19

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
Displays and Alert to Sysadmins when they login or open the app and the server is below the current ESR (5.19).
If user is not a sysadmin or the server is the ESR or above no alert is displayed.

This PR is required in order to merge https://github.com/mattermost/mattermost-mobile/pull/4386

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-25946

#### Checklist
- [x] Includes text changes and localization file updates

### CODE REVIEW

**Comment by enahum** [comment]
@esethna @josephbaylon give it a spin once the build is done

**Comment by esethna** [comment]
@josephbaylon can you link a test server running < v5.19 for testing this?

**Comment by josephbaylon** [comment]
> @josephbaylon can you link a test server running < v5.19 for testing this?
@esethna 
@enahum provided me one last time. Can you please spin up your 5.18 server again and paste here?

**Comment by enahum** [comment]
@josephbaylon @esethna I can, but we need to coordinate as having that server up force me to stop whatever I'm doing...

**Comment by esethna** [comment]
@enahum I can test now

File: assets/base/i18n/en.json [hunk 1]
```diff
@@ -453,8 +453,11 @@
   "mobile.server_link.unreachable_team.error": "This link belongs to a deleted team or to a team to which you do not have access.",
   "mobile.server_ssl.error.text": "The certificate from {host} is not trusted.\n\nPlease contact your System Administrator to resolve the certificate issues and allow connections to this server.",
   "mobile.server_ssl.error.title": "Untrusted Certificate",
+  "mobile.server_upgrade.alert_description": "This server version is unsupported and users will be exposed to compatibility issues that cause crashes or severe bugs breaking core functionality of the app. Upgrading to server version {serverVersion} or later is required.",
```
Comment (enahum) [inline assets/base/i18n/en.json:456]: @esethna updated message

**Pull request merged**